xml/System.Collections.Concurrent/ConcurrentDictionary`2+AlternateLookup`1.xml

```diff
@@ -247,14 +241,9 @@
       </Parameters>
       <Docs>
         <param name="key">The alternate key of the value to get.</param>
-        <param name="actualKey">
-          <para>When this method returns, contains the actual key associated with the alternate key, if the key is found;</para>
-          <para>otherwise, the default value for the type of the key parameter.</para>
-        </param>
-        <param name="value">
-          <para>When this method returns, contains the value associated with the specified key, if the key is found;</para>
-          <para>otherwise, the default value for the type of the value parameter.</para>
-        </param>
+        <param name="actualKey">When this method returns, contains the actual key associated with the alternate key, if the key is found;</para>
+          <para>otherwise, the default value for the type of the key parameter.</param>
+        <param name="value">When this method returns, contains the value associated with the specified key, if the key is found; otherwise, the default value for the type of the value parameter.</param>
```

Comment:
This produced invalid XML and did not correctly eliminate `<para>` and `</para>` tags.
```suggestion
        <param name="actualKey">
            When this method returns, contains the actual key associated with the alternate key, if the key is found;
            otherwise, the default value for the type of the key parameter.
        </param>
        <param name="value">
            When this method returns, contains the value associated with the specified key, if the key is found; otherwise, the default value for the type of the value parameter.
        </param>
```